src/journal-remote/journal-remote.c

```diff
@@ -1415,17 +1416,20 @@ static int parse_argv(int argc, char *argv[]) {
 
                 case ARG_GNUTLS_LOG: {
 #ifdef HAVE_GNUTLS
-                        const char *word, *state;
-                        size_t size;
+                        p = optarg;
+                        for (;;) {
+                                _cleanup_free_ char *word;
 
-                        FOREACH_WORD_SEPARATOR(word, size, optarg, ",", state) {
-                                char *cat;
+                                r = extract_first_word(&p, &word, ",", 0);
+                                if (r < 0) {
+                                        log_error("Failed to parse --gnutls-log: %s", optarg);
+                                        return  -EINVAL;
+                                }
 
-                                cat = strndup(word, size);
-                                if (!cat)
-                                        return log_oom();
+                                if (r == 0)
+                                        break;
 
-                                if (strv_consume(&arg_gnutls_log, cat) < 0)
+                                if (strv_consume(&arg_gnutls_log, word) < 0)
```

Comment:
Oh sorry didn't see the comments from Lennart (didn't get e-mail about them because they were in the commits and not the PR.) Anyways, sorry for the duplicates...